### PR TITLE
public cloud: Rename IPA package to install latest version

### PR DESCRIPTION
**Pull request by asmorodskyi**
Rename IPA package to install latest version

### CODE REVIEW

File: tests/publiccloud/prepare_tools.pm [hunk 1]
```diff
@@ -36,7 +36,7 @@ sub run {
     for my $repo (split(/\s+/, $tools_repo)) {
         zypper_call('ar ' . $repo);
     }
-    zypper_call('--gpg-auto-import-keys -q in python3-ipa python3-ipa-tests git-core ntp');
+    zypper_call('--gpg-auto-import-keys -q in python3-img-proof python3-img-proof-tests git-core ntp');
```
Comment (cfconrad) [inline tests/publiccloud/prepare_tools.pm]: Lets also change the `$default_tools_repos` to  `https://build.suse.de/project/show/Devel:PubCloud:CI`.

Comment (cfconrad) [reply tests/publiccloud/prepare_tools.pm]: oh It is already there :)

**Comment by jlausuch** [comment]
LGTM

**Comment by asmorodskyi** [comment]
this is already part of another bigger PR , closing this one

**Pull request closed without merging**